src/applications/appeals/shared/tests/utils/focus.unit.spec.js

```diff
@@ -77,6 +84,110 @@ describe('focusIssue', () => {
   });
 });
 
+describe('focusFileCard', () => {
+  let focusElementSpy;
+  beforeEach(() => {
+    focusElementSpy = sinon.spy(focusUtils, 'focusElement');
+  });
+
+  afterEach(() => {
+    focusElementSpy.restore();
+  });
+
+  const renderPage = () =>
+    render(
+      <div id="main">
+        <ul className="schemaform-file-list">
+          <li id="root_file_0" className="va-growable-background">
+            <strong id="root_file_name_0">Test1.pdf</strong>
+            <div>32KB</div>
+            <div>
+              <va-button
+                secondary
+                class="delete-upload vads-u-width--auto"
+                label="Delete Test1.pdf"
+                text="Delete file"
+                uswds
```

Comment:
uswds is the default now, including it won't break anything but it isn't necessary to render the v3 version of the component.

Reply:
Oh good to know! Thanks!